modules/migration-DNS-considerations.adoc

```diff
@@ -0,0 +1,51 @@
+. Place a network element such as an application load balancer or a reverse proxy between the clients and the OpenShift Container Platform 4 cluster.
+
+. Update the original application FQDN (`app1.apps.ocp3.example.com`) in the DNS server to return the IP address of the network device.
+
+. Configure the network device to send requests received for the application in the source domain to the load balancer in the target {product-title} 4 cluster domain.
+
+. Create a wildcard DNS record for the `*.apps.ocp3.example.com` domain that points to the IP address of the load balancer of the source cluster.
+
+. Create a specific DNS record for each application that points to the IP address of the network device in front of the target cluster. A specific DNS record has higher priority than a wildcard record, so no conflict arises when the application FQDN is resolved.
+
+Keep in mind the following:
+
+* The network device must terminate all secure TLS connections. If the connections pass through to the {product-title} 4 load balancer, the FQDN of the target application is exposed to the client, and certificate errors occur.
+
+* The applications must not return links referencing the target cluster domain to the clients. Otherwise, parts of the application might not load or work properly.
+
+=== Setting up the target cluster to accept the source DNS domain
+You can set up the target cluster to accept requests for migrated applications in the DNS domain of the source cluster.
+
+For both non-secure (HTTP) and secure (HTTPS) access, do the following:
+
+. Create a route in the application’s project for the host name that applications used in the source cluster:
++
+[source,terminal]
+----
+$ oc expose svc app1-svc  --hostname app1.apps.ocp3.example.com \
+ -n app1-namespace
+----
++
+With this new route in place, any request for that FQDN is accepted and sent to the corresponding application pods.
+In addition, when you migrate the application, another route is created in the target cluster domain (app1.apps.ocp4.example.com). Requests can reach the migrated application using both of these host names.
+
+. Create a specific DNS record for the FQDN in the route host name parameter that points to the IP address of the default load balancer of the target cluster. Specific DNS records take priority over wildcard records.
++
+The FQDN of the application resolves to the load balancer of the target cluster. The default ingress controller router accept requests for that FQDN because a route for that host name is exposed.
+
+For secure HTTPS access, add the following step:
+
+. Replace the x509 certificate of the default ingress controller created during the installation process with a custom certificate that includes the wildcard DNS domains for both the source and target clusters in the Subject Alternative Name (SAN) field. The new certificate is valid for securing connections made using either of the two DNS domains.
```

Comment:
Where is this field located? 

Field names need markup. See OCP guidelines for markup.

No need to add acronym because you never mention it again.